packages/schedule/src/ScheduleContainer.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+
+import { useSchedule, IUseScheduleReturnValue } from './useSchedule';
+
+export interface IScheduleContainerProps {
```

Comment:
This could import and extend an interface from the `useSchedule` hook.

Reply:
I did not create an interface for the `useSchedule` hook arguments because the argument types were inferred by TypeScript. 

I can create the interface for the `useSchedule` arguments so that it's more explicit, and so that `IScheduleContainerProps` can extend from the interface.

Reply:
Let's still create this interface even though it can be inferred. This lets it be used by `ScheduleContainer` and other hooks which want to extend it in the future.

Reply:
I definitely should have included that interface. Updated, thank you for the suggestion 🙏 !